test/test_pipeline/components/test_setup_networks.py

```diff
@@ -0,0 +1,65 @@
+import os
+import sys
+
+import pytest
+
+import torch
+
+from autoPyTorch.pipeline.tabular_classification import TabularClassificationPipeline
+
+
+# Disable
+def blockPrint():
+    sys.stdout = open(os.devnull, 'w')
+
+
+# Restore
+def enablePrint():
```

Comment:
what is this haha

Reply:
I added this as the traditional classifiers had way too many things printed but then I added it to base model I think when the classifiers are fitted. I'll remove them from here

Reply:
oh but this is useless here :P